web/src/pages/index.md

```diff
@@ -7,35 +7,55 @@ title: ECE 477 Team 5 Homepage
+field changes caused by the puck's movement. The density and layout of this grid
+ensure the puck's position is determined with high accuracy, allowing the system
+to provide real-time feedback. For instance, as the puck moves, the individually
+controlled LEDs beneath the surface illuminate its path, creating a mesmerizing
+trail effect that enhances the overall gaming experience.
+
+This grid-based hall effect sensor system not only offers unparalleled precision
+in puck tracking but also enables the microcontroller to interpret rapid changes
+in direction and speed. Such granular detail ensures that dynamic visuals
+generated by the LEDs are always synchronized with the puck's position and
+movement. Players, therefore, experience a highly responsive and visually rich
+gameplay, where the table reacts instantly to every twist and turn of the puck.
+
+Combined with other features like the OLED score display and photoresistor-based
+goal detection, the SAHT offers an intuitive yet futuristic air hockey
+experience. It seamlessly merges traditional gameplay with modern tech
+enhancements, ensuring that both newcomers and seasoned players are treated to
+an unmatched recreational encounter.
 
 ## Project Specific Design Requirements (PSDRs)
 
 The design and development of the smart air hockey table will adhere to the
 following focused design requirements to ensure a successful and captivating
 gameplay experience:
 
-1. PSDR #1 (Hardware):
-   An ability to convert analog hall effect sensor readings into digital signals with custom threshold values using comparators.
-2. PSDR #2 (Hardware):
-   An ability to compress digital readings from a grid of hall effect sensors to simplified row and column position data using logic gates.
-3. PSDR #3 (Hardware):
-   An ability to reliably detect goals scored by monitoring the passage of the puck through the goal using a photoresistor and LED pair.
-4. PSDR #4 (Hardware):
-   An ability to display game score on an OLED display using the SPI protocol.
-5. PSDR #5 (Software):
-   An ability to control a grid of individually-addressable RGB LEDs beneath the playing surface using a serialized PWM communication protocol.
+1. PSDR #1 (Hardware): An ability for the microcontroller to determine the
+   puck's position on the playing surface by interpreting signals from a grid of
+   hall effect switches beneath it, using digital gates to consolidate
+   individual signals into row and column bit vectors.
+2. PSDR #2 (Hardware): An ability for the microcontroller to automatically
+   detect goals scored using an LED and photoresistor pair installed at each
+   goal.
+3. PSDR #3 (Hardware): An ability for the microcontroller to display game score
+   on an OLED display using the SPI protocol.
+4. PSDR #4 (Hardware): An ability for the microcontroller to to control a large
+   grid of individually-addressable RGB LEDs (WS2812B) beneath the playing
+   surface using the WS2812B communication protocol.
```

Comment:
Looks good, I would consider removing the (WS2812B) since we mention the protocol 5 words later.  Especially if we for some reason go to WS2813 or SK6812 or something else that uses the same protocol, the PSDR still applies.